src/count_passenger.c

```diff
@@ -17,6 +17,11 @@ int count_passenger()
     FILE *f;
     char a[10];
     f=fopen("datafolder/input.txt","r");
+    if (f==NULL) 
```

Comment:
f==NULL condition captures any errors in opening input files.
There can be a possibility of empty input file in specified path .
I think,there has to be a check for this too.

Reply:
Empty file will return count as zero. Its handled.